src/constantProductCalculator.ts

```diff
@@ -1,8 +1,25 @@
 import D from "decimal.js";
 
+import { isqrt } from "./isqrt";
 import { Pool } from "./pool";
 import { SwapCalculator } from "./types";
 
+export function getConstantProductMintedLiquidityTokens(
+  addedPrimary: bigint,
+  addedSecondary: bigint,
+  totalPrimary: bigint,
+  totalSecondary: bigint,
+  totalLiquidity: bigint,
+): bigint {
+  if (totalPrimary + totalSecondary === 0n) {
+    return isqrt(addedPrimary * addedSecondary);
+  }
```

Comment:
Here we should lock 1000 lts. Replacing `isqrt(addedPrimary * addedSecondary);` with `isqrt(addedPrimary * addedSecondary) - 1000n;` should do the work